map2loop/deformation_history.py

```diff
@@ -30,7 +30,8 @@ def __init__(self):
         """
         The initialiser for the deformation history. All attributes are defaulted
         """
-        self.minimum_fault_length_to_export = 500.0
+        
+        self.minimum_fault_length_to_export = 0.0
```

Comment:
should this be a parameter that can be initialised?

Reply:
in case it's not user-defined, I think it would be better to automatically calculate the minimum fault length given the scale of the map